ui/installer/VCSA/configs

```diff
@@ -10,3 +10,6 @@ VIC_UI_HOST_URL="NOURL"
 
 # If VIC_UI_HOST_URL starts with "https", enter SHA-1 thumbprint of the web server hosting the vic-ui.zip file
 VIC_UI_HOST_THUMBPRINT=""
```

Comment:
turns out that VC only takes a fingerprint with `:`s. i'll make that clear in the configs file and also possibly let Stuart be aware of it.